cmd/virt-handler/virt-handler.go

```diff
@@ -102,42 +102,36 @@ func main() {
 		panic(err)
 	}
 
-	l, err := labels.Parse(fmt.Sprintf(v1.NodeNameLabel+" in (%s)", *host))
-	if err != nil {
-		panic(err)
-	}
+	// Create all the informers used by virt-handler here
+	informerFactory := kubeinformers.NewKubeInformerFactory(restClient, clientSet)
+	vmOnHostInformer := informerFactory.VmOnHost(*host)
+	domainSharedInformer := informerFactory.CustomInformer("domainInformer", func() cache.SharedIndexInformer {
```

Comment:
Ok, when thinking about this, I don't really see a reason, to move this to a cluster wide factory. It is purely virt-handler and libvirt related code.